js/lib/src/utils/connection.ts

```diff
@@ -17,6 +17,10 @@ export const CLUSTERS: Cluster[] = [
     name: "devnet",
     url: clusterApiUrl("devnet"),
   },
+  {
```

Comment:
I think matches might have been the last contract that was using this util. This function exists in sol-kit already with localnet.